src/main/steps/timed-out/content.ts

```diff
@@ -0,0 +1,22 @@
+import { TranslationFn } from '../../app/controller/GetController';
+
+const en = {
+  title: 'You were signed out to protect your privacy',
+  line1: 'Your application was inactive for more than 20 minutes so you were signed out.',
```

Comment:
Minor comment - do we need a comma (,) after 20 minutes?

Reply:
content has been agreed by Jon - and is used on other services time out pages as well